sympy/polys/multivariate_resultants.py

```diff
@@ -159,7 +159,7 @@ def get_max_degrees(self, polynomial):
 
     def get_dixon_matrix(self, polynomial):
         r"""
-        Construct the Dixon matrix from the coefficients of polynomial
```

Comment:
This is actually the form recommended by [PEP 257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) for one-line docstrings and also for the summary lines of [multi-line docstrings](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).

Reply:
Thanks for the review. Just to be clear, did you mean I should remove the 's' that I added or change the whole docstring according to PEP 257? 
I noticed that none of the docstrings is formatted correctly. Should I change all the docstrings of this class?

Reply:
> Should I change all the docstrings of this class?

That is not necessary, but I think that this one could be left as is and new docstrings should preferably conform to the recommendation (blank line after summary line).

Reply:
@jksuom I made some changes. Let me know if you I think there is anything else I should change.